translate.mjs

```diff
@@ -0,0 +1,37 @@
+import fetch from 'node-fetch';
+import glob from 'glob';
+import * as fs from 'fs';
+const key = process.argv[2];
+const value = process.argv[3];
+
+if (!key || !value) {
+    console.log("Should be 'npm run translate key value'")
+    process.exit(1)
+}
+glob('./site/*.json', {}, (err, files) => {
+  files.forEach(pathName => {
+    const file_content = fs.readFileSync(pathName);
+    const content = JSON.parse(file_content);
+    let languageCode = pathName.slice(7, 9);
+    if (languageCode == 'ee') languageCode = 'et';
```

Comment:
Better to fix the dodgy language code than add hacks to sidestep it!

Reply:
We can sneak it into this PR though right?

Reply:
If you want to get in on the PR sure, but I don't expect him to clean up our mess.